presto-docs/src/main/sphinx/release/release-0.197.rst

```diff
@@ -0,0 +1,65 @@
+=============
+Release 0.197
+=============
+
+General Changes
+---------------
+
+* Fix query scheduling hang when the ``concurrent_lifespans_per_task`` session property is set.
+* Fix query failure when query contains a ``TIMESTAMP`` literal corresponding to a local time
```

Comment:
`query` twice is a bit hard for me to read.  Maybe `Fix failure when a query contains a...`